examples/shard-server.config.example

```diff
@@ -255,6 +255,34 @@ ac_policy: READ_AND_WRITE
 # the listening port of the buildfarm grpc server
 port: 8980
 
+# the publishing port of the buildfarm prometheus metrics
+prometheus_config: {
+  port: 9090
+}
+
+# Publish GRPC metrics to promethesus for each server endpoint. This is used
+# for SLAs at API boundaries. You may want something similar at the proxy level
+# to see frontend/backend performance.
+grpc_metrics: {
+
+  # Whether to enable this metric gathering at all. If disabled, there should
+  # be no related metrics registered in prometheus.
+  enabled: False
+  
+  # These metrics are more expensive to calculate. If disabled, you only
+  # receive core metrics. Core metrics include send/receive totals tagged
+  # with return codes.  No latencies.
+  provide_latency_histograms: False
+  
+  # A status code label will be added to latency histogram. If latency
+  # histogram itself is disabled, this takes no effect. Warning: this will
+  # increase the number of histograms by a factor of actually happened codes
+  # (up to io.grpc.Status.Code values count), which could lead to additional
+  # local memory usage and load on prometheus (storage and memory usage,
+  # query-time complexity)
+  tag_latencies_by_code: False
```

Comment:
You can remove this, now that you put the commented out code into a branch to work on this later.

Reply:
removed